.travis.yml

```diff
@@ -9,109 +9,121 @@ env:
     - export CXX_STANDARD=11
     - export MONOLITH=ON
 
+stages:
+  - "Core tests"
+  - "Additional tests"
+
 jobs:
   include:
-    - name: macOS 10.14, Apple Clang
+    - name: macOS 10.15, Apple Clang
       env:
         - CC=cc
         - CXX=c++
       os: osx
-      osx_image: xcode10.2
+      osx_image: xcode12
+      stage: "Core tests"
       addons:
         homebrew:
           packages:
             - libomp
-          update: true
+          update: false
 
-    - name: macOS 10.14, llvm 8
+    - name: macOS 10.15, Apple Clang, non-monolithic
       env:
-        - CC=/usr/local/opt/llvm@8/bin/clang
-        - CXX=/usr/local/opt/llvm@8/bin/clang++
+        - CC=cc
+        - CXX=c++
       os: osx
-      osx_image: xcode10.2
+      osx_image: xcode12
+      stage: "Additional tests"
       addons:
         homebrew:
           packages:
-            - llvm@8
             - libomp
           update: true
+      script:
+        - $CXX --version
+        - mkdir build_non_monolith && cd "$_"
+        - cmake -DNETWORKIT_MONOLITH=OFF -DNETWORKIT_BUILD_TESTS=ON -DCMAKE_BUILD_TYPE=Debug ..
+        - make -j2
+        - ctest -V
```

Comment:
Actually, this block is not needed, there is already a block `script_cpp_only` already defined below that does the same thing. You could move that block here, and add `MONOLITH=OFF` in `env`.